frostsnap_coordinator/src/firmware_upgrade.rs

```diff
@@ -64,10 +69,20 @@ impl UiProtocol for FirmwareUpgradeProtocol {
         }
     }
 
-    fn process_upgrade_mode_ack(&mut self, from: DeviceId) {
-        if !self.state.confirmations.contains(&from) {
-            self.state.confirmations.push(from);
-            self.emit_state()
+    fn process_comms_message(&mut self, from: DeviceId, message: CommsMisc) {
+        match message {
+            CommsMisc::AckUpgradeMode => {
+                if !self.state.confirmations.contains(&from) {
+                    self.state.confirmations.push(from);
+                    self.emit_state()
+                }
+            }
+            _msg => {
+                error!(
```

Comment:
Maybe overthinking but could upgrading be more robust to changes if we just ignore other messages here?

Reply:
isn't that what we are doing by just logging them?

Reply:
woops! yes, i was thinking of device panics